modules/042-kube-dns/openapi/config-values.yaml

```
@@ -91,3 +91,25 @@ properties:
     items:
       type: string
       pattern: '^[0-9a-zA-Z\.-]+$'
+  stsPodsHostsAppender:
+    type: object
+    description: |
+      Settings for the StatefulSet pods hosts appender webhook used to render additional FQDNs in `/etc/hosts` for Pods with `spec.subdomain` when `clusterDomainAliases` are set.
+    properties:
+      initContainerRequests:
+        type: object
+        description: |
+          Resource requests for the init container injected into Pods by the webhook.
+        properties:
+          cpu:
+            oneOf:
+              - type: number
+                format: int
+              - type: string
+            pattern: "^[0-9]+m?$"
```

```suggestion
            pattern: "^[0-9]+m?$"
            default: '10m'
```
It seems to me that a clear description of default values would be more helpful.

---

Removed settings for requests